### ISSUE TITLE
Build Prerequisite: Meson

### ISSUE THREAD

**Issue by mergen3107** [koreader/koreader#12678]
@benoit-pierre 
I think build prerequisite for `meson` should revised. 

In the doc, it is asking to install meson along all other packages. 

However, in Ubuntu MATE 2024.04.1 as well as Linux Mint Debian Edition 6 what is installed through `sudo apt install meson` is of version 1.0.1 (or something like that). 

I had to install meson via python's pip install instead on LMDE and the make a soft link to /usr/bin/meson from that python-installed meson. Don't remember what I did in Ubuntu MATE

**Comment by benoit-pierre** [comment]
Isn't Ubuntu MATE 2024.04.1 based on noble (24.04LTS)? Which means the available meson version should be 1.3.2.

**Comment by benoit-pierre** [comment]
Debian Bookworm does indeed provide only meson 1.0.1. ¯\\_(ツ)_/¯ The curse of those eternally outdated distributions… The build instructions at the top does specify the minimum version of each requirement.

**Comment by mergen3107** [comment]
Interesting, I'll go double check later. 
This was about in June maybe when I first did it in MATE (python meson install, that is).

**Comment by benoit-pierre** [comment]
1.5.1 is included in `bookworm-backports`.

**Comment by mergen3107** [comment]
I probably should go and check what's other versions are :D 

The fact that KOReader built doesn't mean I met all of them, right? :D

**Comment by mergen3107** [comment]
Oh, that's cool! I should check it out, otherwise it would be a hassle to keep it updated

**Comment by benoit-pierre** [comment]
The top-level makefiles also check the compilers provide C11 & C++17 support.

**Comment by mergen3107** [comment]
Thanks again! I double-checked other packages that require specific version and they are fine.
I uninstalled the old meson `pip3 uninstall --break-system-packages meson` (the same way I installed it).

Here is how I installed meson from bookworm-backports (the repository was already included on LMDE 6):
```
sudo apt install meson/bookworm-backports
```
Full instructions here: https://wiki.debian.org/Backports
